CodeEditModules/Modules/AppPreferences/src/Media.xcassets/Settings Image - Auto.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Don't use images for this. Make a custom SwiftUI View instead.

Also do we have rights to use Apple's background images?

Reply:
We have. Apple's background image can be used, while the other parts are all drawn by myself.

Reply:
Point me to a resource that explicitly states it is free to use. I'm pretty sure it is Apple's intellectual property.

Reply:
The Apple software (including Boot ROM code), any third party software, documentation, interfaces, content, fonts and any data accompanying this License whether preinstalled on Apple-branded hardware, on disk, in read only memory, on any other media or in any other form (collectively the “Apple Software”) are licensed, not sold, to you by Apple Inc. (“Apple”) for use only under the terms of this License. Apple and/or Apple’s licensors retain ownership of the Apple Software itself and reserve all rights not expressly granted to you.

Reply:
All images are from the sketch plugin provided by Apple

Reply:
We may use these resources on a non-profit basis.

Reply:
Yeah and this does not include taking the image and putting it in your own software project.

Reply:
Ok, I'll change the image.

Reply:
Fixed